pkgs/by-name/op/ophcrack/package.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  stdenv,
+  libsForQt5,
+  fetchurl,
+  fetchpatch,
+  pkg-config,
+  openssl,
+  expat,
+  enableGui ? true,
+}:
+stdenv.mkDerivation {
+
+  pname = "ophcrack";
+  version = "3.8.0";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/ophcrack/files/ophcrack/3.8.0/ophcrack-3.8.0.tar.bz2";
+    hash = "sha256-BIpt9XmDo6WjGsfE7BLfFqpJ5lKilnbZPU75WdUK7uA=";
+  };
+
+  patches = [
+    (fetchpatch {
+      url = "https://salsa.debian.org/pkg-security-team/ophcrack/-/raw/c60118b40802e1162dcebfe5f881cf973b2334d3/debian/patches/fix_spelling_error.diff";
+      hash = "sha256-Fc044hTU4Mtdym+HukGAwGzaLm7aVzV9KpvHvFUG2Sc=";
+    })
+    (fetchpatch {
+      url = "https://salsa.debian.org/pkg-security-team/ophcrack/-/raw/e19d993a7dbf131d13128366e2aac270a685befc/debian/patches/qmake_crossbuild.diff";
+      hash = "sha256-sOKXOBpAYGLacU6IxjRzy3HCnGm4DFowDL2qP+DzG8M=";
+    })
+  ];
+
+  nativeBuildInputs = [ pkg-config ] ++ lib.optional enableGui libsForQt5.wrapQtAppsHook;
+  buildInputs = [ openssl ] ++ (if enableGui then [ libsForQt5.qtcharts ] else [ expat ]);
+
+  configureFlags =
+    [ "--with-libssl=yes" ]
+    ++ (
+      if enableGui then
+        [
+          "--enable-gui"
+          "--with-qt5charts"
+        ]
+      else
+        [ "--disable-gui" ]
+    );
+
+  postinstall = lib.optional enableGui ''
+    wrapProgram $out/bin/ophcrack \
+      --prefix PATH : "${libsForQt5.qtbase}/bin"
```

Comment:
@Tochiaha can you explain why this is necessary ?

Reply:
with this nix expression their wont be any need for the postinstall. i was experiencing this [error](https://discourse.nixos.org/t/appimage-run-fails-loading-qt-platform-plugin-xcb/29293?u=tochi4h4) before this solution.

Reply:
how do you reproduce it, just run it ? I don't get any errors without this `postinstall`

I am on gnome wayland

Reply:
i don't get any errors without the postInstall now.

Reply:
if you don't have any questions, feel free to cherry pick this to your PR